src/append/rolling_file/policy/compound/trigger/time.rs

```diff
@@ -236,14 +250,22 @@ impl TimeTrigger {
             let weekday = current.weekday().num_days_from_monday() as i64; // Monday is the first day of the week
             let time = Local.with_ymd_and_hms(year, month, day, 0, 0, 0).unwrap();
             let increment = if modulate { n - week0 % n } else { n };
-            return time + Duration::weeks(increment) - Duration::days(weekday);
+            let dur = Duration::try_weeks(increment).ok_or(
+                TimeTriggerIntervalError::OutOfBounds("Weeks".to_owned(), interval),
+            )? - Duration::try_days(weekday).ok_or(
+                TimeTriggerIntervalError::OutOfBounds("Days".to_owned(), interval),
```

Comment:
It can lead to the misunderstanding, as the `weekday` is not set by the user. Lets remove this ok_or

Reply:
done